style.css

```diff
@@ -138,28 +200,158 @@ div p {
   flex-direction: column;
 }
 
-.heading h3 {
-  width: 327px;
-  color: #1c1a19;
-  font-family: 'Crete Round', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 40px;
-  height: 50px;
-  margin-top: 120px;
-}
-
-.heading #line {
-  width: 327px;
-  margin-left: inherit;
-  margin-top: 0;
-  margin-bottom: 50px;
-}
-
-.main-project img {
-  width: 327px;
-  height: 237px;
-  top: 307px;
+@media (min-width: 768px) {
+  .work {
+    grid-template-columns: auto 29% 29% 29% 0;
+    padding-bottom: 120px;
+    row-gap: 1.6%;
+    column-gap: 1.6%;
+    padding-top: 10%;
+    grid-template-areas:
+      ". first second second ."
+      ". main main main ."
+      ". one two three ."
+      ". four five six .";
+  }
+
+  .heading {
+    font-family: "Crete Round", sans-serif;
+    width: 315px;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    grid-area: first;
+    margin-left: 60px;
+    color: #091e42;
+  }
+
+  #line {
+    border: 1px solid #9c9494;
+    align-self: center;
+    grid-area: second;
+    margin: auto 0;
+  }
+
+  .desktop-project {
+    grid-area: main;
+    width: 55%;
+    height: auto;
+    margin-top: 8%;
+  }
+
+  .main-project {
+    display: none;
+  }
+
+  .text {
+    margin-left: 58%;
+    grid-area: main;
+    margin-top: 8%;
+  }
+
+  .project .button {
+    display: none;
+  }
+
+  #one {
+    grid-area: one;
+    background-image: url(images/one.png);
+    border: 2px solid #091e42;
+  }
+
+  #one > * {
+    display: none;
+  }
+
+  #one .button {
+    display: block;
+    right: 0;
+    left: 0;
+  }
+
+  #two {
+    grid-area: two;
+    background-image: url(images/two.png);
+  }
+
+  .mobile-version {
+    display: none;
+  }
+
+  #three {
+    grid-area: three;
+    background-image: url(images/three.png);
+  }
+
+  #three h2 {
+    height: 42px;
+  }
+
+  #four {
+    grid-area: four;
+    background-image: url(images/background-img.png);
+  }
+
+  #five {
+    grid-area: five;
+    background-image: url(images/two.png);
+  }
+
+  #six {
+    grid-area: six;
+    background-image: url(images/three.png);
+  }
+
+  #six h2 {
+    height: 42px;
+  }
+
+  .project {
+    max-width: 100%;
+    margin-bottom: 0;
+    padding-bottom: 0;
+    justify-content: flex-end;
+  }
+}
+
+@media (max-width: 767px) {
+  .desktop-version {
+    display: none;
+  }
+}
+
+@media (max-width: 767px) {
+  .heading {
+    width: 327px;
+    color: #1c1a19;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    height: 50px;
+    margin-top: 120px;
+  }
+}
+
+@media (max-width: 767px) {
+  #line {
+    width: 327px;
+    margin-left: inherit;
+    margin-top: 0;
+    margin-bottom: 50px;
+  }
+}
```

Comment:
- [ ] Kindly fix the line as the reviewer mentioned. It shouldn't go through the text.

![image](https://user-images.githubusercontent.com/93630700/206732641-226650f9-0672-49bd-876d-938acd7a7a4c.png)